File: sdkwork-ai-sdk-typescript/src/types/assistant-list-response.ts
import type { AssistantResponse } from './assistant-response';

/** OpenAI assistant list response */
export interface AssistantListResponse {
  /** Object type */
  object?: string;
  /** Assistant list */
  data?: AssistantResponse[];
  /** Whether more data exists */
  has_more?: boolean;
  /** First item id */
  first_id?: string;
  /** Last item id */
  last_id?: string;
}

File: sdkwork-ai-sdk-go/types/assistant_list_response.go
package types

// OpenAI assistant list response
type AssistantListResponse struct {
	Object string `json:"object"`
	Data []AssistantResponse `json:"data"`
	HasMore bool `json:"has_more"`
	FirstId string `json:"first_id"`
	LastId string `json:"last_id"`
}
